arts/telescope.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+
+class Telescope:
+    """
+    Class with Telescope properties.
+
+    """
+
+    def __init__(self):
```

Comment:
Maybe add the name, location, etc to the constructor arguments?

Reply:
Good idea. Will do.

Reply:
Done.